core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -67,7 +67,7 @@ object ApiVersionsRequestTest {
     serverProperties.put("unstable.feature.versions.enable", "false")
     List(ClusterConfig.defaultBuilder()
       .setTypes(java.util.Collections.singleton(Type.ZK))
-      .setMetadataVersion(MetadataVersion.IBP_3_7_IV4)
+      .setMetadataVersion(MetadataVersion.IBP_3_8_IV0)
```

Comment:
I can change it to `latestProduction` here. I can't change it below because `@ClusterTest` requires an actual constant as an annotation, and `latestProduction()` is a function rather than a constant.